src/libraries/Microsoft.PowerFx.Interpreter/RecalcEngineResolver.cs

```diff
@@ -23,6 +25,19 @@ public RecalcEngineResolver(RecalcEngine parent, PowerFxConfig powerFxConfig)
             _powerFxConfig = powerFxConfig;
         }
 
+        public override IEnumerable<TexlFunction> LookupFunctions(DPath theNamespace, string name, bool localeInvariant = false)
+        {
+            if (theNamespace.IsRoot)
+            {
+                if (_parent._customFuncs.TryGetValue(name, out var func))
+                {
+                    return new TexlFunction[] { func };
```

Comment:
This is interesting, we could theoretically support multiple overloads from UDFs at some point.